lib/src/material/material_desktop_controls.dart

```diff
@@ -485,40 +485,37 @@ class _MaterialDesktopControlsState extends State<MaterialDesktopControls>
   void _onExpandCollapse() {
     setState(() {
       notifier.hideStuff = true;
+    });
 
-      chewieController.toggleFullScreen();
-      _showAfterExpandCollapseTimer =
-          Timer(const Duration(milliseconds: 300), () {
-        setState(() {
-          _cancelAndRestartTimer();
-        });
+    chewieController.toggleFullScreen();
+
+    _showAfterExpandCollapseTimer =
+        Timer(const Duration(milliseconds: 300), () {
+      setState(() {
+        _cancelAndRestartTimer();
       });
     });
   }
 
   void _playPause() {
-    final isFinished = _latestValue.position >= _latestValue.duration;
-
-    setState(() {
-      if (controller.value.isPlaying) {
+    if (controller.value.isPlaying) {
+      setState(() {
         notifier.hideStuff = false;
-        _hideTimer?.cancel();
-        controller.pause();
-      } else {
-        _cancelAndRestartTimer();
+      });
 
-        if (!controller.value.isInitialized) {
-          controller.initialize().then((_) {
-            controller.play();
-          });
-        } else {
-          if (isFinished) {
```

Comment:
The `Duration` should be set back to `zero` if the video is finished player and `controller.value.isLooping` is `true`.

Reply:
Even with the `VideoPlayerController` `play()` seeking to `zero`? (this may be an update)

<img width="707" alt="image" src="https://github.com/fluttercommunity/chewie/assets/120418414/49c6c926-46b3-4c76-aded-e4db16763ceb">

Reply:
I think I see your point. When hitting play, it will automatically start from the beginning if already finished according to the `VideoController`'s implementation.

In that case, add comments instead, describing this behavior so that other devs are aware that `VideoController` behaves this way.